src/tests/ftest/osa/offline_drain.py

```diff
@@ -123,14 +124,16 @@ def run_offline_drain_test(self, num_pool, data=False, oclass=None, pool_fillup=
                     self.print_and_assert_on_rebuild_failure(output)
                     total_space_after_reintegration = self.pool.get_total_space(refresh=True)
                     self.assertGreater(total_space_after_reintegration, total_space_after_drain,
-                                       "Expected total space after reintegration is less than drain")
+                                       "Expected total space after reintegration \
+                                        is less than drain")
```

Comment:
You can probably also do this, line the " up four spaces to the right of the previous indentation.
```suggestion
                        "Expected total space after reintegration is less than drain")
```

Reply:
This is more helpful as it allows for grepping and easier reading of the code.

Reply:
Done.

Reply:
Hmm. Why I am getting flake8/pylint issues? (under intend).